std/node/_fs/_fs_chmod.ts

```diff
@@ -13,20 +13,9 @@ export function chmod(
   mode: string | number,
   callback: CallbackWithError
 ): void {
-  new Promise(async (resolve, reject) => {
-    try {
-      await Deno.chmod(path, getResolvedMode(mode));
-      resolve();
-    } catch (err) {
-      reject(err);
-    }
-  })
-    .then(() => {
-      callback();
-    })
-    .catch((err) => {
-      callback(err);
-    });
+  Deno.chmod(path, getResolvedMode(mode))
+    .then(() => callback)
```

Comment:
this seems wrong. Shouldn't it be `.then(callback)` ?

Reply:
This functions , does'n provide any callback arguments while promise resolved,

Just one Error type argument on catch

According to [node document](https://nodejs.org/api/fs.html)

Reply:
So it should be `.then(() => callback())`?

Reply:
right , was my hurry , done 👍